src/features/lightspeed/playbookGeneration.ts

```diff
@@ -22,10 +23,23 @@ async function openNewPlaybookEditor(playbook: string) {
   const editBuilder = (textEdit: any) => {
     textEdit.insert(new vscode.Position(0, 0), String(playbook));
   };
-  return await editor.edit(editBuilder, {
+
+  await editor.edit(editBuilder, {
     undoStopBefore: true,
     undoStopAfter: false,
   });
+
+  lightSpeedManager.contentMatchesProvider.suggestionDetails = [
+    {
+      suggestionId: generationId,
+      suggestion: playbook,
```

Comment:
I think we should instead loop on all the tasks of the playbook and add an entry in this list. This would:
- allow us to continue to use the regular contentMatch end-point
- and reduce the complexity of the code base  by avoiding the `isPlaybook` parameter, on having just one branch in the code logic.

Reply:
So I prefer to sending a playbook to WCA.
Because Lightspeed (WCA) gives a playbook as a suggestion, I think it is more natural to set the whole playbook to the `suggestion` parameter to call contentMatch. 

This PR also parses the playbook (in lines 226-239 of contentMatchesWebView.ts) since the response provides one dimensional array of content match object per task.   However, I saw WCA generated a playbook that contained two plays. If we send an array of tasks collected from those two plays, some structural information is lost.


I guess probably the current WCA implementation is reusing the code match code implemetned for single-/multi-task inference suggestion.  So it is very likely that we get the same results even if we send a list of tasks instead of a playbook.

Reply:
For the record: I will the code to send a playbook as the input. If issues arise with it, I will update the code.